sapi/apache2handler/php_functions.stub.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+function apache_lookup_uri(string $filename): object|false {}
+
+function virtual(string $uri): bool {}
+
+function apache_request_headers(): array {}
+
+function getallheaders(): array {}
+
+function apache_response_headers(): array {}
+
+function apache_note(string $note_name, string $note_value = ""): string|false {}
+
+function apache_setenv(string $variable, string $value, bool $walk_to_top = false): bool {}
+
+function apache_getenv(string $value, bool $walk_to_top = false): string|false {}
+
+function apache_get_version(): string {}
```

Comment:
According to the code, this can also return `false`. Not sure if that ever could happen in practise, though.

Reply:
Better to include that then, just in case